docs/design/colocation/Overview.md

```diff
@@ -0,0 +1,456 @@
+```
+
+Enable node level colocation by setting lable volcano.sh/oversubscription=true and volcano.sh/colocation=true.
+
+```
+$ kubectl label node $node volcano.sh/oversubscription=true # replace $node with real node name in your kubernetes cluster.
+
+$ kubectl label node $node volcano.sh/colocation=true # replace $node with real node name in your kubernetes cluster.
+```
+
+### CPU burst tutorial
+
+This example will demonstrate how to use CPU burst and what benefit it can bring.
+
+#### Enable cpu burst by set pod annotation
+
+Run a nginx deployment with cluster ip service exposed, pod with annotations `volcano.sh/enable-quota-burst: "true"` indicates pod cpu
+
+burst is enabled.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+      annotations: 
+        volcano.sh/enable-quota-burst: "true" # pod enabled cpu burst 
+    spec:
+      containers:
+      - name: container-1
+        image: nginx:latest
+        resources:
+          limits:
+            cpu: "2"
+          requests:
+            cpu: "1"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  namespace: default
+  labels:
+    app: nginx
+spec:
+  selector:
+    app: nginx
+  ports:
+    - name: http
+      targetPort: 80
+      port: 80
+      protocol: TCP
+  type: ClusterIP
+```
+
+#### Perform stress
+
+Perform stress with `wrk` to nginx app.
+
+```shell
+wrk -H "Accept-Encoding: deflate, gzip" -t 2 -c 8 -d 120  --latency --timeout 2s http://$(kubectl get svc nginx -o jsonpath='{.spec.clusterIP}')
+```
+
+#### Check cpu burst
+
+Check container's cpu throttle state of pod, we can see that `nr_bursts` and `burst_time` is not 0, `nr_throttled` and `throttled_time` is a small value, which indicates pod has used burst cpu quota.
+
+```shell
+$ cat /sys/fs/cgroup/cpu/kubepods/burstable/podd2988e14-83bc-4d3d-931a-59f8a3174396/cpu.stat # replace nginx pod uid in your kubernetes cluster.
+nr_periods 1210
+nr_throttled 9
+throttled_time 193613865
+nr_bursts 448
+burst_time 6543701690
+```
+
+If we set pod's annotations `volcano.sh/enable-quota-burst=false`(disable pod cpu burst) and perform another stress, `nr_throttled` and `throttled_time` will be a relatively large value, which indicates pod cpu is throttled strictly, `nr_bursts` and `burst_time` is 0, indicates that pod cpu burst not happened.
+
+```shell
+$ cat /sys/fs/cgroup/cpu/kubepods/burstable/podeeb542c6-b667-4da4-9ac9-86ced4e93fbb/cpu.stat #replace nginx pod uid in your kubernetes cluster.
+nr_periods 1210
+nr_throttled 488
+throttled_time 10125826283
+nr_bursts 0
+burst_time 0
+```
+
+#### Limitation
+
+CPU burst relies on capabilities provided by the linux kernel, this feature only works with host upstream linux kernel >=5.14 and some Linux Distribution like OpenEuler 22.03 SP2 or higher version.
```

Comment:
This function has specific requirements on the underlying operating system. If the operating system does not meet the requirements, can I disable this function?

Reply:
Yeah it can be disabled and you can find the guide in `Advanced settings` part.